package.json

```diff
@@ -14,7 +14,9 @@
     "docs:build": "rm -rf docs && typedoc --out docs --exclude '**/index.ts' src",
     "docs:publish": "npm run docs:build && git checkout gh-pages && cp -R docs/* ./ && rm -rf docs && git add . && git commit -m 'Update docs' && git push origin gh-pages && git checkout master",
     "lint": "tslint -p tsconfig.json",
-    "prepublishOnly": "npm run test && npm run lint && npm run build"
+    "prepublishOnly": "npm run test && npm run lint && npm run build",
+    "e2e": "cypress run",
+    "e2e:watch": "cypress open"
```

Comment:
We should probably npm run e2e to prepublishOnly, but we don't know that charlie can run these tests... they run in electron headlessly so they do work but.... FUD

Reply:
I found the tests deterministic without needing much "wait" complexity... that is until I started trying in electron.  - 

I'd have to spend another hour or two to get electron to a good state, and I think there is value even if Charlie can't run the tests for us... so putting off electron.